.pylintrc

```diff
@@ -8,6 +8,9 @@ ignore=migrations,south_migrations
 # regex matches against base names, not paths.
 ignore-patterns=.*_pb2.py
 
+# Consider that variables that start with a "_" may not be used.
+dummy-variables-rgx=_.*$
+
```

Comment:
je ne suis pas trop d'accord avec cette stratégie...

pour moi une variable commencant par _  signifie une "variable privée", c'est tout autre chose...
convention classique: https://stackoverflow.com/questions/9998348/python-leading-underscore-variables

pour le problème que tu essaies de résoudre ici, je l'ai résolu dans ma branche vgrange/importer simplement comme ceci par exemple: `df, _ = result`

Reply:
Effectivement, je croyais que le préfixe avec `_` était une pratique standard ; apparemment, c'est utilisé mais peu fréquent : https://stackoverflow.com/questions/11486148/unused-variable-naming-in-python
Du coup, je suis d'accord pour remplacer les variables inutilisées par `_`.